pages/dev/governance/community-pool-proposals.mdx

```diff
@@ -1,8 +1,8 @@
-# Community Grants Proposals
+# Community Pool Proposals
```

Comment:
this should not be under developers, as anyone can/should be able to submit these. 
maybe create its own top level page for governance? or put under resources?

Reply:
Added to the resources category.